C++/C++.sublime-syntax

```diff
@@ -71,7 +71,7 @@ variables:
   other_keywords: 'typedef|nullptr|{{visibility_modifiers}}|static_assert|sizeof|using|typeid|alignof|alignas|namespace|template'
   modifiers: '{{storage_classes}}|{{type_qualifier}}|{{compiler_directive}}|{{constant_expression_specifiers}}'
   non_angle_brackets: '(?=<<|<=)'
-  module_identifier: '(\:?[{{identifier}}.])(\:?[{{identifier}}.])?'
+  module_identifier: '[{{identifier}}\.]+'
```

Comment:
> The module name consists of one or more identifiers separated by dots (for example: mymodule, mymodule.mysubmodule, mymodule2...). Dots have no intrinsic meaning, however they are used informally to represent hierarchy. 

see: https://en.cppreference.com/w/cpp/language/modules

Wouldn't the statement result in the following pattern?

```suggestion
  module_identifier: '{{identifier}}(?:\.{{identifier}})*'
```

Reply:
You're right and I just checked MSVC too.  Module names cannot start with a dot.  I pushed the change, as well as some other changes to provide syntax highlighting when trying to create invalid module names.